src/main/kotlin/greedy/Camping.kt

```diff
@@ -0,0 +1,17 @@
+package greedy
+
+import java.io.BufferedReader
+import java.io.InputStreamReader
+
+fun main() = with(BufferedReader(InputStreamReader(System.`in`))) {
+    var cnt = 1
+    while (true) {
+        val (l, p, v) = readLine().split(' ').map { it.toInt() }
+        if (l == 0 && p == 0 && v == 0) {
```

Comment:
```suggestion
        if (l == 0) {
```
문제 조건에서  (1 < L < P < V)으로 주어졌기 떄문에 L, P, V 어느 하나만 0인지 검사해서 0일 경우 반복문을 종료시킬 수 있을 것 같습니다!

Reply:
감사합니다!